packages/blade/.storybook/react/preview.js

```diff
@@ -38,12 +40,63 @@ export const parameters = {
   },
   docs: {
     theme,
+    components: {
+      summary: styled.summary`
+        font-family: ${theme.fontBase};
+        color: ${theme.textColor};
+        font-weight: normal;
+        cursor: pointer;
+      `,
+      li: styled.li`
+        :not(:first-child) {
+          padding-top: 16px;
+        }
+        font-size: 14px;
+
+        & :not(pre) > code {
+          line-height: 1;
+          margin: 0 2px;
+          padding: 3px 5px;
+          white-space: nowrap;
+          border-radius: 3px;
+          font-size: 13px;
+          border: 1px solid #eeeeee;
+          background-color: #f8f8f8;
+        }
```

Comment:
It removed the styles of `code` component inside list so I copy-pasted the styles again in this.

In storybook overrides, at certain places like these you would think, "Hey why are we doing this?" and in most such cases the answer is "that is the only way to make it work"

Reply:
since it got overridden because you added these styles there might be other components whose style might have broken too and we might have to add those styles?

Reply:
Yupp!! So I checked `p` and it wasn't overridden. Definitely there might be something that is overriden but don't see anything broken in docs right now. We can fix it here if something looks strange in docs.

Also even with overriden styles, it doesn't completely go out of theme. It has some styles by default. `code` in particular was looking a bit strange.